client/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "client",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
```

Comment:
I feel that the following 3 testing dependencies need to go into the "devDependencies" of package.json, since it is most likely only used during development. So leaving the "dependencies" section only contains the package that is needed for production.

Reply:
good point - I will move these